plugins/inputs/openstack/README.md

```diff
@@ -75,6 +75,13 @@ your requirements. This will help with load and cardinality as well.
   username = "admin"
   password = "password"
 
+  ## AllowReauth should be set to true if you grant permission for Gophercloud to
+  ## cache your credentials in memory, and to allow Gophercloud to attempt to
+  ## re-authenticate automatically if/when your token expires.  If you set it to
+  ## false, it will not cache these settings, but re-authentication will not be
+  ## possible.  This setting defaults to false.
+  # allow_reauth = false
+
```

Comment:
I'm not sure if we really need to guide this with an option... Are there cases where the token does not expire or where the current behavior is expected and _wanted_?

Reply:
Thanks for the quick response, the default expiry of Openstack token is one hour however there is a provision to extend it further but it's not advised to do so for security aspects as this uses admin level access.
What I have observed in our environment is for frequency like 30 min interval(which is ideal to monitor some services in Openstack), the token is expired after two sets of data collection and throws an error of unauthorised access.

Reply:
Thanks for the PR. This sounds similar to how the k8s token is expiring tokens every hour (see https://github.com/influxdata/telegraf/issues/11267). This does sound more like a bug fix and one that should be the default behavior. @srebhan thoughts?

Reply:
I agree. I would not wrap it with an option but "do the right thing" (tm) ;-) without forcing the user to enable the right behavior. TL;DR: Remove the option and do the processing as if `allow_reauth=true`.